rotkehlchen/db/filtering.py

```diff
@@ -1153,6 +1164,18 @@ def make(
                 operator=ignored_assets_filter_params[0],
                 values=ignored_assets_filter_params[1],
             ))
+        if lps_handling != NftLpHandling.ALL_NFTS:
```

Comment:
It seems `NftLpHandling.ALL_NFTS` does not do anything. So just set `lps_handling` to have a default of None.
So you have...
```python
if lps_handling is not None:
    ...
```

Reply:
It's kind of the same so I think it's fine. Having None without a reason complicates things